Core/GDCore/IDE/Dialogs/LayoutEditorCanvas/LayoutEditorCanvasOptions.cpp

```diff
@@ -21,7 +21,9 @@ LayoutEditorCanvasOptions::LayoutEditorCanvasOptions()
       gridG(180),
       gridB(255),
       zoomFactor(1),
-      windowMask(false) {}
+      windowMask(false),
+      xValue(400),

```

Comment:
This could mean that a game with another size than 800x600 is not properly centered at the beginning - could be confusing and not a super user experience even if not a huge deal.

Can we make another boolean variable that is false by default to indicate that the positions are not valid (and we can initialize them to 0) and set to true as soon as it's valid (i.e: changed by the user)?

Reply:
I added a check if `viewPositionValid` is true(false by default), only then `scrollTo` function is called, hope this resolves it